src/client/views/AdminPanel.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from "react";
+import { IPattern } from "../utils/types";
+import patternService from "../services/pattern";
+import PatternCard from "../components/PatternCard";
+import { Link, useNavigate } from "react-router-dom";
+
+interface AdminPanelProps {}
+
+const AdminPanel = (props: AdminPanelProps) => {
+	const navigate = useNavigate();
+	const [patterns, setPatterns] = React.useState<IPattern[]>([]);
+
+	const handleDelete = (id: string) => {
+		patternService
+			.destroyPattern(id)
+			.then(() => navigate("/patterns/admin"))
+			.catch();
+	};
+
+	useEffect(() => {
+		patternService.getAllPatterns().then((data) => setPatterns(data));
+		// .catch((e) => Toast.error(e.message));
```

Comment:
I assume the places you have these are possibly needed down the line? If so, they're fine to keep, but if not, they should be cleaned up!

Reply:
Yeah this is going to be implemented later with more comprehensive error handling and the UX pass